extensions/keycloak-authorization/deployment/src/test/java/io/quarkus/keycloak/pep/test/KeycloakAuthorizationOfflineEnforcerTest.java

```diff
@@ -0,0 +1,109 @@
+package io.quarkus.keycloak.pep.test;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import jakarta.inject.Inject;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.keycloak.representations.AccessTokenResponse;
+
+import io.quarkus.keycloak.pep.runtime.PolicyEnforcerResolver;
+import io.quarkus.test.QuarkusUnitTest;
+import io.restassured.RestAssured;
+
+public class KeycloakAuthorizationOfflineEnforcerTest {
+
+    @RegisterExtension
+    final static QuarkusUnitTest app = new QuarkusUnitTest()
+            .withApplicationRoot(jar -> jar
+                    .addClasses(AdminResource.class, PublicResource.class, UserResource.class)
+                    .addAsResource("offline-enforcer-application.properties", "application.properties")
+                    .addAsResource("quarkus-realm.json"));
+
+    @Inject
+    PolicyEnforcerResolver resolver;
+
+    @Inject
+    @ConfigProperty(name = "keycloak.url")
+    String keycloakUrl;
+
+    @Test
+    public void testNoPathMatcherRemoteCalls() {
+
+        // first test Keycloak policy enforcer works
+        testPublicResource();
+        testAccessUserResource();
+        testAccessAdminResource();
+
+        // as we can't test whether there were remote calls without extensive logging
+        // here we test whether expected decisions are made
+        testIsPathEnforcementDisabled();
+    }
+
+    @Test
+    public void testConflictingPaths() {
+        // path '/api/conflict/*' is both enforced and disabled
+        // therefore we expect path to be disabled to stay on the safe side
+        var defaultEnforcer = resolver.getPolicyEnforcer(null);
+        assertTrue(defaultEnforcer.isPathEnforcementDisabled("/api/conflict/stuff"));
```

Comment:
@michalvavrik How is it a safe side ? What if a user made a mistake and it is meant to be enforced ? I think if the same path has multiple policies then the stronger policy must prevail.

Reply:
It should not be disabled and I believe the way to fix is to add the check I proposed earlier

Reply:
@sberyozkin Hmm, true. The comment is miserable, I made mistake. I'm sorry.

I don't understand how you want me to change default Keycloak behavior. Am I supposed to write my own path matching mechanism that will prefer stronger policy? I think this PR must not change Keycloak path matching logic.